frontend_tests/node_tests/stream_data.js

```diff
@@ -438,6 +438,12 @@ zrequire('marked', 'third/marked/lib/marked');
     assert(!stream_data.is_subscribed('Canada'));
     assert(!stream_data.get_sub('Canada'));
     assert(!stream_data.get_sub_by_id(canada.stream_id));
+
+    blueslip.warn = function (msg) {
+        assert.deepStrictEqual(msg, 'Failed to delete stream EEXIST');
```

Comment:
Is there any reason you have to use `deepStrictEqual` instead of just `equal` here? (Same for below)

Reply:
I prefer strict assert just personal preference and good practice.